Samples/iOS15-SwiftUI/iOS15-SwiftUI/Info.plist

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
`m`: If you decide to generate this `Info.plist` also via `xcodgen` I would say we should remove this from git aswell to stay consistent with absent Xcode projects.